### ISSUE TITLE
Change "Legal Structure" to "Business Structure" everywhere in Navigator

### ISSUE THREAD

**Issue by hinzed1127** [hinzed1127/pivotal-to-gh-importer#280]
This issue was imported from Pivotal Tracker.

  | Field | Value |
  | --- | --- |
  | Story ID | [#183652493](https://www.pivotaltracker.com/story/show/183652493) |
  | State | accepted |
  | Requested By | Anne LoVerso |
  | Owned By | Anne LoVerso |
  | Estimate | 0 |
  | Priority | p3 - Low |
  | Created At | Oct 27, 2022 |
  | Accepted At | Dec 1, 2022 |
      
## User Stories

As a Navigator user, I do not want to see references to "legal structures" in the system with preference to "business structures" so that I have better context for my circumstances.

## Acceptance Criteria

Given I am a Navigator user
when accessing any area of the system where 'legal structure' was referenced
then the corresponding content refers to 'business structure' instead of 'legal structure'
Include a list of changes for Content

## Implementation Details

"Legal Structure" -> "Business Structure"
"legal structure" -> "business structure"

## Analytics

- Has the SEO improved after implementation?
- Compare link clicks and search terms before and after release 
- Need capture the release date for comparison

## Comments
@anneloverso_nj @jessica2I @ceciliaramirez1 @kellieadamson I've refined this story and we prioritized it earlier today together likely for next sprint (14). (dillonsullivan - Nov 10, 2022)

@dillonsullivan Per above, this "kind of" been prioritized, but is unpointed. Should we give it the pre-grooming label? (ej3 - Nov 17, 2022)

@anneloverso_nj @mikewolfd since this has sort have been prioritized above the set of items that still need to be estimated, might it be possible for the dev team to point this off-cycle of grooming, like in a dev standup? (dillonsullivan - Nov 17, 2022)

@dillonsullivan Sure, we can definitely do that if necessary. Do you think it's likely we'll get to this item before our regular grooming next Monday? If so, we can point it whenever you like. However, it seems like we still have a lot of work ahead of it, so if we won't get it to it before Monday, we could also just do it during regular grooming? (Anne LoVerso - Nov 17, 2022)

Commit by Anne LoVerso
https://github.com/newjersey/navigator.business.nj.gov/commit/bee9fb1263dc7a1aef8656bdfb3146f7dc11ffcc
navigator.business.nj.gov

feat: [#183652493] change legal structure -> business structure in the content (Anne LoVerso - Nov 29, 2022)

https://docs.google.com/document/d/1HDN2pwYRrHAXdrlhqNTQhZdcH3PT4bjDQO5FsHsNsLM/edit?usp=sharing

@kellieadamson @ceciliaramirez1 @jessica2I ready for review, and here is the doc with the list of places where the content has changed (Anne LoVerso - Nov 29, 2022)

@anneloverso_nj @ceciliaramirez1 @jessica2I looks like all but the config items on the list were changed. attached two screenshots that show where i'm still seeing legal structure.. (kellieadamson - Nov 30, 2022)

@kellieadamson those were in fact changed (you can see it in dev) but the old content is still in the CMS because of the active draft on that file https://dev.navigator.business.nj.gov/mgmt/cms#/workflow

once that draft is approved then the changes I made will be merged into the config (Anne LoVerso - Nov 30, 2022)

**Content** review set to **pass**

all good! (kellieadamson - Nov 30, 2022)

Two searches for 'business structure' after December 1st. There was a third search prior to Dec. 1st. 'Legal structure' was search twice pre-release and once post-release.  (dillonsullivan - Feb 3, 2023)
